src/amo/pages/UserProfileEdit/utils.js

```diff
@@ -0,0 +1,3 @@
+import { MZA_LAUNCH_DATETIME } from 'amo/constants';
```

Comment:
It's missing a `@flow` annotation.

Reply:
Given the constant is only used here, I think it doesn't strictly need to be in `amo/constants` either. Here would be fine. There is a banner to show somewhere, though, according to @eviljeff's comment earlier - if the banner is not on the user profile "edit" page,the `utils.js` file should be moved somewhere else, probably in `amo/utils` (in a `fxa.js` file so that it's clear what this is about and easy to find when we'll do some cleanups).

Reply:
(the banner will be on the user profile edit page)